pkg/printers/internalversion/printers.go

```diff
@@ -1713,7 +1714,11 @@ func printCertificateSigningRequest(obj *certificates.CertificateSigningRequest,
 	if err != nil {
 		return nil, err
 	}
-	row.Cells = append(row.Cells, obj.Name, translateTimestampSince(obj.CreationTimestamp), obj.Spec.Username, status)
+	signerName := "<none>"
+	if obj.Spec.SignerName == "" {
```

Comment:
You have this backwards:

`if len(obj.Spec.SignerName) != 0 {`

Reply:
yeah, also check for nil:
```
if obj.Spec.SignerName != nil && len(obj.Spec.SignerName) > 0 {
```

Reply:
Fixed, added a test for this case and fixed the failing tests 😄